pkg/coord/local/clocal.go

```diff
@@ -159,10 +173,25 @@ func (lc *LocalCoordinator) AlterDistributionAttach(ctx context.Context, id stri
 		if len(r.DistributionKey) != len(ds.ColTypes) {
 			return fmt.Errorf("cannot attach relation %v to this dataspace: number of column mismatch", r.Name)
 		}
+		if !r.ReplicatedRelation && len(r.ColumnSequenceMapping) > 0 {
+			return spqrerror.Newf(spqrerror.SPQR_INVALID_REQUEST, "sequence are supported for replicated relations only")
```

Comment:
In the documentation, there is no mention of the term "replicated relationships".
TODO add docs